apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -151,6 +154,24 @@ defmodule SiteWeb.CustomerSupportController do
     MIME.from_path(filename) in @allowed_attachment_types
   end
 
+  # Errors we'd expect to see from reCAPTCHA assuming no bugs in the library.
+  # See: https://developers.google.com/recaptcha/docs/verify#error_code_reference
+  @expected_recaptcha_errors [
+    :challenge_failed,
+    :invalid_input_response,
+    :timeout_or_duplicate,
+    # https://github.com/samueljseay/recaptcha/issues/58
+    :"invalid-input-response"
+  ]
+
+  @spec validate_recaptcha(map) :: :ok | String.t()
+  defp validate_recaptcha(%{"recaptcha_response" => response}) do
+    case Recaptcha.verify(response) do
+      {:ok, _} -> :ok
+      {:error, [error]} when error in @expected_recaptcha_errors -> "recaptcha"
```

Comment:
@digitalcora let's log the original error at level `warn`, because I'd like to set up a Splunk alert in case we start getting repeated errors here.

Reply:
Okay, although I'm not sure of the usefulness, since this case covers just the errors that are part of the normal operation of a properly-implemented reCAPTCHA (hence _expected_ errors).